src/generators/image_generators/stable_diffusion_with_loras.py

```diff
@@ -152,9 +150,8 @@ def request_scene_image_generation(
             prompt=server_settings.quest_background_image_prompt,
             negative_prompt=server_settings.quest_background_image_negative_prompt,
             template_vars={
-                "genre": game_state.genre or "Adventure",
```

Comment:
question: is dropping `genre` here intentional?

Reply:
Yup; Genre was discussed to be somewhat duplicative to the combination of "tone" and "background", and thus was removed for now